Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Editor/Resources/BMS_Forge_Editor/NetworkManagerTemplate.txt

```diff
@@ -69,6 +69,12 @@ namespace BeardedManStudios.Forge.Networking.Unity
 		[Obsolete("Use Instantiate>:[i]:< instead, its shorter and easier to type out ;)")]
 		public >:[i]:<Behavior Instantiate>:[i]:<NetworkObject(int index = 0, Vector3? position = null, Quaternion? rotation = null, bool sendTransform = true)
 		{
+            if (>:[i]:<NetworkObject.Length <= index)
+            {
+                Debug.Log("Prefab(s) missing for: >:[i]:<. Add them at the NetworkManager prefab.");
+                return null;
+            }   
+
```

Comment:
Indentation looks to be wrong please use tabs instead of spaces as per the contributing doc.

Reply:
Should be fixed now!